packages/mdc-textfield/README.md

```diff
@@ -390,8 +390,38 @@ Returns an object representing the native text input element, with a similar API
 
 Returns the idle outline element's computed style value of the given css property `propertyName`. The vanilla implementation achieves this via `getComputedStyle(...).getPropertyValue(propertyName)`.
 
+### `MDCTextFieldLabelAdapter`
```

Comment:
It looks like only the `MDCTextFieldOutlineAdapter` methods were listed under `MDCTextFieldAdapter`.

I thought it would be easier to read if we split them up into individual sub-adapter interfaces, but I can move all the methods under `MDCTextFieldAdapter` if you prefer.

Reply:
Right now it's inconsistently split into some of the sub-adapter interfaces, with other methods left directly under MDCTextFieldAdapter, with no indication that the sub-adapter interfaces also need to be implemented at the same time.

I think for purposes of wrapper libraries, as well as for this PR, keeping them all in one place is fine. Wrapper libraries will need to wrap all of the methods regardless.

Reply:
Done